src/Card.jsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react'
+import './Card.css'
+
+export default function Card(title, text, isBookmarked) {
```

Comment:
We use destructuring assignment to get the properties `title`, `text` and `isBookmarked` from the `props` object we get as a parameter from React. It's either:
```js
export default function Card(props) {
    const {title, text, isBookmarked} = props
```
Or:
```suggestion
export default function Card({title, text, isBookmarked}) {
```
And that is the same as if we'd write it like this: `const {title, text, isBookmarked} = props`